episodes/07-manipulating-MARC-data-advanced.md

```diff
@@ -64,6 +82,33 @@ This is possible using the function called, Build New Field.
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+:::::::::::::::::::::::::::::::::::::::::::::::::: callout
+
+## Beware the Build New Field Function
+In the example above, if your 856 contains other subfields such as $3, then these will not be retained because you are only building a new 856 that replaces the current one. Also the build new field works on the first iteration of the field. When working in particular with the MARC field 856, if you need to build a new field using all of the MARC fields 856 in your record, you can use the syntax [x] as in {856$u[x]}. This will ensure all of the MARC fields in the records will undergo the build operation you specify.
+
+::::::::::::::::::::::::::::::::::::::::: 
+
+## Advanced Subfield Edit Functions
+There are 3 advanced Edit Subfield features that are extremely useful. These are append, prepend, and change subfield. These aren't regular expressions but special characters built into the MarcEditor Edit Subfield tool.
+
+- Prepend data to a subfield: Special character ^b
+- Change subfield character: Special character ^c
+- Append data to a subfield: Special character ^e
+
+:::::::::::::::::::::::::::::::::::::::  checklist
+
+## Let's add the proxy to the 856$u using the prepend special character ^b
```

Comment:
Do we need to add an instruction to perform a special undo here? Or is this to catch the remaining 856 fields without the proxy? If so, maybe we can specify this explicitly.